ember-cli-build.js

```diff
@@ -33,8 +33,10 @@ module.exports = function () {
     'ember-cli-babel': {
       includePolyfill: true,
     },
-    babel: {
-      optional: ['es7.decorators']
+    // need to use babel6 key here until
+    // we can upgrade to ember-cli@2.13
+    babel6: {
```

Comment:
Yeah, sadly using babel with older ember-cli makes it try to use these babel 6 plugins for ember-cli's own internal transpilation (which is still using babel@5).